opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -249,6 +249,12 @@ message Span {
   // An optional final status for this span. Semantically when Status isn't set, it means
   // span's status code is unset, i.e. assume STATUS_CODE_UNSET (code = 0).
   Status status = 15;
+
+  // parent_span_is_remote identifies whether a span's parent is remote or not.
+  // This helps determine whether a span can be considered an entry-point span.
```

Comment:
I have never heard "entry-point span". The term I heard before for this was "local root span" (e.g. opentelemetry-specification#366).

Reply:
The term "entry-point span" was used in the OTEP; does it matter if this change matches the terminology used in the OTEP?

Reply:
In general the OTEP wording is preferable, but here I would definitely prefer "local root span" as you will find many other issues around the OTel repos with that keyword. Alternatively, I would also be OK if you just added

```suggestion
  // This helps determine whether a span can be considered an entry-point span (also known as "local root span").
```